import-service/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "import-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
+  "main": "handler.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "dependencies": {
+    "@types/jest": "^26.0.15",
```

Comment:
I suggest to move all packages except `csv-parser` to devDependencies to reduce bundled lambda size.

Reply:
@AndreyS55, thanks a lot, I will fix this